src/styles/_mixin.scss

```diff
@@ -74,3 +74,26 @@
   border-radius: $border-radius;
   border-color: $border-color;
 }
+
+// TextComponent Addition
+@mixin input-color($background-color, $border-width, $border-color) {
```

Comment:
Redundant code here, you can reuse something here including CSS cascading rule.

Reply:
Oh yeah I meant to delete input-color, because input-colors below it replaces it